### ISSUE TITLE
add voicing when T2S of titles is not reliable

### ISSUE THREAD

**Issue by murata2makoto** [w3c/epub-specs#1288]
In EPUB Accessibility Techniques, recommend the use of "voicing" (a link relationship) when T2S for titles and authors is not reliable and the target audience is likely to be confused by incorrect T2S.

**Comment by dauwhe** [comment]
>  recommend the use of "voiding"

I presume you mean "voicing" from https://w3c.github.io/publ-epub-revision/epub32/spec/epub-packages.html#sec-voicing

**Comment by mattgarrish** [comment]
This looks like it would fall under meeting 3.1.6: https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-pronunciation.html

**Comment by avneeshsingh** [comment]
@murata2makoto are you talking about having an audio file to play the title of the publication if TTS is not speaking it in the right way?
It would be helpful if you can explain the use case more.
My assumption is that this would be useful in Media Overlays book, and the EReader would need to implement additional capability to play title audio. This means that it would be more of a specialized feature used by some reading systems.
Looking forward to more info.

**Comment by mattgarrish** [comment]
To recommend this as a technique we should also be able to show that there is support (i.e., hold ourselves to the same standard of techniques being [accessibility supported](https://www.w3.org/TR/WCAG21/#dfn-accessibility-supported)).

Is there broad adoption of the voicing property to present audio? If not, it's too early to say that following a technique would actually help meet the related WCAG success criterion.

**Comment by avneeshsingh** [comment]
One way to handle it is to develop best practices instead of a formal technique. And up grade it to a formal technique when there is wider support.

**Comment by murata2makoto** [comment]
I will discuss in the JDC TC this weekend and get back to you.

**Comment by mattgarrish** [comment]
> develop best practices instead of a formal technique

Yes, we should try to follow an "incubation" model like we're following for new features in the main specification. Once we're comfortable that these practices will be widely beneficial (and this includes the other issues like report dates and icons), then we can go the final step of formalizing them as techniques.

WCAG only adds techniques after they've proven themselves in the wild. We shouldn't perpetuate the IDPF model of pushing things into standards in the hope that that will lead to adoption.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-05-13-epub-a11y#section2) on 2021-05-13

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 2. Add voicing when TTS of titles is not reliable 

_See github issue [#1288](https://github.com/w3c/epub-specs/issues/1288)._




**Avneesh Singh:** summarizing discussion up to now in issue, we don't think many RS support this feature right now  
… we propose adding this to best practices, and if we see uptake, we then add to techniques document at that point  

**Matt Garrish:** key is that we need to show implementation  
… proves practical benefit over theoretical use  

**George Kerscher:** would this be in metadata or body?  

**Matt Garrish:** in the package metadata  
… so you can use the refines attribute with link element to point to audio file of title/author names  

**Avneesh Singh:** so we will wait for mokoto's feedback  

</details>

**Comment by murata2makoto** [comment]
I find that the next meeting the JDC TC is not this weekend but next weekend.  Sorry!

**Comment by murata2makoto** [comment]
We agree not to mention this in EPUB Accessibility Techniques due to the lack of wide adoption.

**Comment by mattgarrish** [comment]
This might be worth taking up in the community group if you want to improve support. There likely needs to be more information for reading system devs on how and when to implement this feature first.
